backend/src/server/routes/v1/project-router.ts

```diff
@@ -511,4 +512,52 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
       return { serviceTokenData };
     }
   });
+
+  server.route({
+    method: "GET",
+    url: "/id",
```

Comment:
This feels quite misleading @sheensantoscapadngan 

I would recommend a better name would be /workspace/slug/:slugName rather than as `/workspace/id` as i had very less idea what it meant

Reply:
yeah this one was supposed to just get the ID of the project